README.md

```diff
@@ -0,0 +1,60 @@
+
+# govuk-diff-pages
```

Comment:
We should use the template in https://github.com/alphagov/styleguides/blob/master/use-of-READMEs.md here.